homeassistant/components/launch_library/strings.json

```diff
@@ -1,10 +1,6 @@
 {
   "config": {
-    "step": {
-      "user": {
-        "description": "Do you want to configure the Launch Library?"
```

Comment:
This is a single step config flow, with no parameters.

Wouldn't it show an empty confirmation dialog now?

Reply:
Yes, so would the `iss` integration where you asked me to change it. It also doesn't have any configuration parameters. Or am i misunderstanding something?

Reply:
> so would the iss integration where you asked me to change it.

The ISS integration had a field in that review.

Reply:
Yes, that was moved to the `OptionsFlow`, per your comment

Reply:
Right... 🤷‍♂️ 

You've lost me now. This integration does not ask for anything, in those cases we put in a text to compensate for that; thus removing it here, isn't correct.

Reply:
Okay, now I understand. Should I also add it back in for the `iss` integration again, as it also does not have any parameters in the config flow? 

You asked me to remove it and the move it to the options flow in these comments https://github.com/home-assistant/core/pull/64987#discussion_r795546619 and https://github.com/home-assistant/core/pull/64987#discussion_r795547626. This is way i am confused why you now are asking me not to